playbooks/common/openshift-cluster/upgrades/upgrade.yml

```diff
@@ -203,7 +215,7 @@
     command: >
       {{ openshift.common.admin_binary }} manage-node {{ openshift.common.hostname | lower }} --schedulable=true
     delegate_to: "{{ groups.oo_first_master.0 }}"
-    when: inventory_hostname in groups.oo_nodes_to_config and openshift.node.schedulable | bool
```

Comment:
Maybe they had overriden the master default schedulability in their inventory, then manually set it unschedulable, and were expecting that to be preserved. I'll follow up on the bug asking for more info and explaining what we've done here.